frontend/src/views/SecretOverviewPage/components/CreateSecretForm/CreateSecretForm.tsx

```diff
@@ -183,13 +183,20 @@ export const CreateSecretForm = ({
                       isChecked={field.value}
                       onCheckedChange={field.onChange}
                       id={`secret-input-${env.slug}`}
+                      className="!justify-start"
                     >
-                      {env.name}
-                      {getSecretByKey(env.slug, newSecretKey) && (
-                        <Tooltip content="Secret exists. Will be overwritten">
-                          <FontAwesomeIcon icon={faWarning} className="ml-1 text-yellow-400" />
-                        </Tooltip>
-                      )}
+                      <span className="flex w-full flex-row items-center justify-start whitespace-pre-wrap">
+                        <span title={env.name} className="truncate">
+                          {env.name}
+                        </span>
+                        <span>
+                          {getSecretByKey(env.slug, newSecretKey) && (
+                            <Tooltip content="Secret exists. Will be overwritten">
```

Comment:
Nit: I would probably phrase it slightly different, like "Secret already exists, and it will be overwritten." 

Additionally, you could try adding a `max-w-[150px]` to the Tooltip to somewhat midigate the overflow issue.

Reply:
Updating the width made it better, but not completely resolved it. But better than before